test/functional/test_framework/test_node.py

```diff
@@ -113,6 +113,8 @@ def __init__(self, i, datadir, *, chain, rpchost, timewait, timeout_factor, bitc
 
         if self.version_is_at_least(190000):
             self.args.append("-logthreadnames")
+        if self.version_is_at_least(209900):
```

Comment:
needs rebase

Reply:
Thanks! Rebased! :)

Reply:
I tried to say that this is a silent merge conflict. The version needs to be adjusted with the rebase as well

Reply:
Oh, it should be bumped to `219900`?